### ISSUE TITLE
Namespace for V2 is incorrect?

### ISSUE THREAD

**Issue by gunturaf** [devopsfaith/krakend-cors#19]
When I use https://github.com/devopsfaith/krakend-config-migrator to migrate my configuration for v2, it changed the `github_com/devopsfaith/krakend-cors` namespace into `security/cors` , and this probably broke the CORS functionality because this line https://github.com/devopsfaith/krakend-cors/blob/9e903f85f6681d4651442372baca4c2e4aedf0b7/cors.go#L10 still refers to the old namespace.

Happy to contribute to this, but I wonder if this also happened to other Lura opensource plugins too in the wake of recent v2 release?

**Comment by taik0** [comment]
Hi @gunturaf 

The namespace in the packages will not change.
We introduced an alias system in lura so anyone building a new gateway can set specific keys to it's own configuration.

Check [here](https://github.com/luraproject/lura/blob/master/config/config.go#L297-L307) in lura and [here](https://github.com/devopsfaith/krakend-ce/blob/73603aa4c18bc66cf357f3838de40f9a54ae5e95/cmd/krakend-ce/main.go#L63-L109) in KrakenD-CE to see how it's managed.

In other words, if you don't change the namespace to the new one, the configuration will work as always.

**Comment by gunturaf** [comment]
alright thank you for the explanation, so I just need to populate that public variable just like it was in KrakenD-CE right?

**Comment by alombarte** [comment]
@gunturaf the contents of the krakend-cors object remain the same. Populate them as documented in KrakenD-CE.
